app/controllers/admin/attachments_controller.rb

```diff
@@ -18,22 +18,18 @@ def order
     redirect_to attachable_attachments_path(attachable), notice: "Attachments re-ordered"
   end
 
```

Comment:
I think that's just the bulk update controller. The Attachments one continues to use the get_layout method

Reply:
Sorry: I mean, the method `AttachmentsController#get_layout` needs to be changed. I just picked a random line in this file because the method hasn't changed yet, so it's not in the PR diff.

The `get_layout` method currently looks like this:

https://github.com/alphagov/whitehall/blob/17484c28c14b47c164b5fca0ae89afbe61ccb519/app/controllers/admin/attachments_controller.rb#L88-L96

i.e. the now-migrated actions (line 89) are still flagged behind `preview_design_system?(next_release: true)` – but they shouldn't be any more.

Reply:
oh right sorry, was being daft. Have sorted